src/components/cc-beta/cc-beta.types.d.ts

```diff
@@ -1 +1 @@
-type PositionType = 'top-left' | 'bottom-left' | 'top-right' | 'bottom-right';
+export type PositionType = 'top-left' | 'bottom-left' | 'top-right' | 'bottom-right';
```

Comment:
issue: this should be in another commit

:arrow_right: this change is available in my proposition branch: https://github.com/CleverCloud/clever-components/tree/typecheck-i18n--hubert